python-test.py

```diff
@@ -1,11 +1,11 @@
 print ("This is a tttsting code to check error catching")
 
-def simple_fn(range):
-    if range > 0:
```

Comment:
Add a condition to check if seed equals 1 in the if statement for more explicit control flow.